hr_timesheet_task_stage/views/account_analytic_line.xml

```diff
@@ -17,15 +17,15 @@
                     tabindex="-1"
                     type="object"
                     icon="fa-folder-o"
-                    attrs="{'invisible': ['|', ('is_task_closed', '=', True), ('task_id', '=', False)]}"
+                    invisible="is_task_closed == True or task_id == False"
```

Comment:
`invisible="is_task_closed or not task_id"` is sufficient

Reply:
THNX for review. Changed.